packages/app/src/components/UserSettingsProvider.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useState, useEffect, createContext } from 'react'
+import PropTypes from 'prop-types'
+import { useDataEngine } from '@dhis2/app-runtime'
+
+export const userSettingsQuery = {
+    resource: 'userSettings',
+    params: {
+        key: ['keyDbLocale', 'keyUiLocale', 'keyAnalysisDisplayProperty'],
```

Comment:
Is `keyDbLocale` used by the app? based on what is in `extractUserSettings` it doesn't seem so.

Reply:
Removed.